### PR TITLE
Move some changes in from boztalay/sites

### PR DESCRIPTION
**Pull request by benmathia**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,224 +1,266 @@
-# sites ![Platform](https://img.shields.io/badge/Platform-macOS%20|%20ubuntu-blue.svg) ![Server](https://img.shields.io/badge/Server-nginx-brightgreen.svg) ![App](https://img.shields.io/badge/App-python%20|%20flask%20|%20gunicorn-red.svg)
```
Comment (christianbator) [inline README.md:1]: You didn't like my badges 😢

File: README.md [hunk 2]
```diff
@@ -1,224 +1,266 @@
...
+
+Set up your server with the following commands, substituting your domain for
+`{host}`.
 
 **1. Clone the repo**
+
 ```
-git clone git@github.com:christianbator/sites.git
-cd sites
+[local]$ git clone git@github.com:boztalay/sites.git
```
Comment (christianbator) [inline README.md]: nmc/Sites.git

Probs everywhere

File: README.md (hunk 1 above)
Comment (benmathia) [reply README.md:1]: sry they made the markdown gross

File: README.md (hunk 2 above)
Comment (benmathia) [reply README.md]: oof

File: README.md [hunk 3]
```diff
@@ -1,224 +1,266 @@
...
-The site design is simple and inflexible, but there are already plenty of tools for managing complex sites. I wanted to post photos to an https site from my laptop and phone, and I wanted to serve them as static content.
+Caveats
+-------
+
+The site design is simple and inflexible, but there are already plenty of tools
+for managing complex sites. I wanted to post photos to an https site from my
+laptop and phone, and I wanted to serve them as static content.
 
-That said, feel free to fork the project and mess around with the `template` directory - there are css files in `template/content/styles`. Restructuring the HTML requires code changes in `app/post.py`.
+That said, *feel free to fork* the project and mess around with the `template`
```
Comment (christianbator) [inline README.md]: Point taken, maybe italics is a little ragey for the general public

File: scripts/ios_shortcut_input.py [hunk 4]
```diff
@@ -0,0 +1,28 @@
+#
+# ios_shortcut_input.py
```
Comment (christianbator) [inline scripts/ios_shortcut_input.py]: Can we rename this to `list_sites.py` or something? :)

File: README.md (hunk 3 above)
Comment (benmathia) [reply README.md]: You're a little ragey for the general public

File: scripts/ios_shortcut_input.py (hunk 4 above)
Comment (benmathia) [reply scripts/ios_shortcut_input.py]: Hmm, `gimme_those_site_keys.py`?

Comment (christianbator) [reply scripts/ios_shortcut_input.py]: `gimme_dem`

**Review by christianbator — APPROVED** [review]

**Comment by benmathia** [comment]
boom merged

**Pull request merged**